app/query_objects/cafe_queries.rb

```diff
@@ -47,6 +47,12 @@ def self.between(start, stop)
       includes(:user)
   end
 
+  def self.between_by_start(start, stop)
+    CafeShift.where('start BETWEEN ? AND ?', start, stop).
+      order(start: :asc, pass: :asc).
```

Comment:
Since this seems to be the convention in this specific file I will skip changing this. It, however, undoubtedly looks nicer with the dot before the method...